src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -474,6 +477,68 @@
             throw new InvalidRequest(ORG_NOT_ACTIVE);
         }
 
-        return mfaStatusService.updateOrgMfaStatus(mfaUpdateRequest,organisation);
+        return mfaStatusService.updateOrgMfaStatus(mfaUpdateRequest, organisation);
+    }
+
+    @ApiOperation(
+            value = "Review (Accept or Reject) an Organisation's registered PBAs ",
+            notes = "**IDAM Roles to access API** : \n prd-admin",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "The requested PBAs have been successfully Updated",
+                    response = NewUserResponse.class

```

Comment:
Please change the response and also include success,partial success in swagger documentation

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/4dd5cfc6d344a2dbee4d78a4dfc175452660cb07